lib/commands/pathdetect.mjs

```diff
@@ -0,0 +1,48 @@
+export function isSystemDirectory(dirPath) {
+    // Normalize the directory path to lower case and ensure it ends with a slash
+    dirPath = dirPath.toLowerCase().replace(/\\+/g, '/');
+    if (!dirPath.endsWith('/')) {
+      dirPath += '/';
+    console.log(dirPath)
```

Comment:
Is this meant to be just debug output, or were you thinking this should stay in our output?